google/ggrc-core
# GGRC-4903 epic/object review acl

# Dependencies

This PR is `on hold` until the dependencies are merged:
- [x] #8332
- [ ]  #8427 - need to be merged together


# Issue description
- Update acl propagation rules for Reviewer
- Make objects reviewable
- remove old implementation (mixins)

# Steps to test the changes
1) Run integration tests.
2) Check propagation rules
Reviewer -> Review RU 
Reviewer -> Reviewable R 
Reviewer -> Reviewable R -> People R
Reviewer -> Reviewable R -> Document/Url R
Reviewer -> Reviewable R -> Comment R

Reviewable Reader -> Review R
Reviewable Editor -> Review RU



# Solution description
- Update propagation config tree. Update Global roles to work with review.
- Add reviewable to :
Control
Program
Regulation
Objective
Clause
Contract
Policy
Risk
Standard
Threat
Requirement
- Remove "has_object_state" mixin

# Sanity checklist




# PR Review checklist
